### PR TITLE
use sockets instead of ports

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: unicoremc/tests/test_config_manager.py [hunk 1]
```diff
@@ -91,3 +110,6 @@ def test_write_cms_nginx_configs(self):
         self.assertTrue('cms.za.qa.ffl.unicore.io' in data)
         self.assertTrue('unicore_cms_django_ffl_za-access.log' in data)
         self.assertTrue('unicore_cms_django_ffl_za-error.log' in data)
+        print data
+        print socket_path
```
Comment (smn) [inline unicoremc/tests/test_config_manager.py]: stray?

File: project/templates/configs/cms.nginx.conf [hunk 2]
```diff
@@ -28,7 +28,7 @@ server {
         proxy_redirect off;
         proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header X-Scheme $scheme;
-        proxy_pass http://127.0.0.1:8803; # TODO: either map country to ports or use sockets cc: @frozbyte
+        proxy_pass unix:{{socket_path}} timeout=0;
```
Comment (froztbyte) [inline project/templates/configs/cms.nginx.conf]: I think take `timeout=0` out here, not convinced it'll work across the board for this statement. seems like it should be used in `upstream { }` definitions only

File: project/templates/configs/frontend.nginx.conf [hunk 3]
```diff
@@ -21,7 +21,7 @@ server {
         proxy_redirect off;
         proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header X-Scheme $scheme;
-        proxy_pass http://127.0.0.1:8703; # TODO: either map country to ports or use sockets cc: @frozbyte
+        proxy_pass unix:{{socket_path}} timeout=0;
```
Comment (froztbyte) [inline project/templates/configs/frontend.nginx.conf]: same here

**Comment by miltontony** [comment]
@smn @froztbyte +1 ?

**Comment by froztbyte** [comment]
:+1:

**Pull request merged**